Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.py

```diff
@@ -2,289 +2,808 @@
+    result = CommandResults(
+        readable_output=hr_data
+    )
+    return_results(result)
+
+
+def get_group(args: dict, client, IdentityStoreId: str) -> None:
+    """
+    Retrieve group information from the AWS IAM Identity Center based on the provided arguments.
+
+    Args:
+        args: A dictionary containing information required to identify the group to be retrieved.
+        client: The client object used to interact with the Identity Store service.
+        IdentityStoreId: The ID of the Identity Store where the group is registered.
+    """
+    response_id = get_groupId_by_displayName(args, client, IdentityStoreId)
+    response = client.describe_group(
+        IdentityStoreId=IdentityStoreId,
+        GroupId=response_id
+    )
+    hr_data = {
+        'DisplayName': response.get('DisplayName'),
+        'GroupId': response.get('GroupId')
+    }
+    response.pop('ResponseMetadata', None)
+    human_readable = tableToMarkdown('AWS IAM Identity Center Groups', hr_data, removeNull=True)
+    result = CommandResults(
+        outputs_prefix=PREFIX_GROUP,
+        readable_output=human_readable,
+        outputs_key_field='GroupId',
+        outputs=response
+    )
+    return_results(result)
+
+
+def list_groups_for_user(args: dict, client, IdentityStoreId: str) -> None:
+    """
+    List groups associated with a user from the AWS IAM Identity Center based on the provided arguments.
+
+    Args:
+        args: A dictionary containing information required to identify the user
+        and optional parameters such as 'limit' and 'nextToken'.
+        client: The client object used to interact with the IAM Identity Center service.
+        IdentityStoreId: The ID of the Identity Store from which groups are listed for the user.
+    """
+    hr_data = []
+    context_data: Dict[str, Any] = {}
+    user_id = get_userId_by_username(args, client, IdentityStoreId)
+    kwargs = {
+        'IdentityStoreId': IdentityStoreId,
+        'MemberId': {
+            'UserId': user_id
+        },
+        'MaxResults': get_limit(args),
+        'NextToken': args.get('nextToken')
+    }
+    kwargs = remove_empty_elements(kwargs)
+    response = client.list_group_memberships_for_member(**kwargs)
+    groups: list = []
+    for group in response.get('GroupMemberships', []):
+        hr_data.append({
+            'UserID': user_id,
+            'GroupID': group.get('GroupId'),
+            'MembershipID': group.get('MembershipId')
+        })
+        groups.append({
+            'GroupId': group.get('GroupId'),
+            'MembershipId': group.get('MembershipId')
+        })
+
+    context_data['UserId'] = user_id
+    context_data['GroupsUserNextToken'] = response.get('NextToken')
+    last_context = demisto.context()
+    last_users = last_context.get('AWS', {}).get('IAMIdentityCenter', {}).get('User', {})
+    last_group_memberships: list = []
+    if not isinstance(last_users, list):
```

Comment:
please extract this logic to a different function if it very similar to list_group_membership please use the same function for both.